tests/make-backup-and-restore-iso/runtest.sh

```diff
@@ -60,9 +60,12 @@ check_and_submit_rear_log() {
     rlFileSubmit $path "rear-$1.log"
 }
 
+ROOT_DISK=$(df -hT | grep /$ | awk '{print $1}')
```

Comment:
If this work reliably enough then fine, but device naming in linux is not stable.